relay-server/src/endpoints/project_configs.rs

```diff
@@ -172,12 +176,9 @@ fn get_project_configs(
                         );
                     };
                 }
-                Ok(None) => {
+                None => {
                     pending.push(project_key);
                 }
-                Err(_) => {
-                    configs.insert(project_key, None);
```

Comment:
This error has been eliminated completely from the interface. The project cache should populate with an errored state, and we should go through `Some` above.